### PR TITLE
Add note about stale read

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Add note about stale read

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6518, https://github.com/pingcap/docs-cn/pull/6525
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: as-of-timestamp.md [hunk 1]
```diff
@@ -32,7 +36,13 @@ Here are some examples of the `AS OF TIMESTAMP` clause:
...
 - `AS OF TIMESTAMP TIDB_BOUNDED_STALENESS(NOW() - INTERVAL 20 SECOND, NOW())`: Tells TiDB to read the data as new as possible within the time range of 20 seconds ago to the present.
 
-Note that in addition to specifying a timestamp, the most common use of the `AS OF TIMESTAMP` clause is to read data that is several seconds old. If this approach is used, it is recommended to read historical data older than 5 seconds.
+> **Note:**
+>
+> In addition to specifying a timestamp, the most common use of the `AS OF TIMESTAMP` clause is to read data that is several seconds old. If this approach is used, it is recommended to read historical data older than 5 seconds.
+>
+> You need to deploy the NTP service for your TiDB and PD nodes when you use Stale Read. This avoids the situation where the specified timestamp of TiDB goes ahead of the latest TSO distribution progress (such as a timestamp several seconds ahead), or goes ahead of the timestamp of GC safe point. When the specified timestamp goes beyond the service scope, TiDB returns an error or waits for the transaction to commit.
```
Comment (NingLin-P) [inline as-of-timestamp.md:43]:
```suggestion
> You need to deploy the NTP service for your TiDB and PD nodes when you use Stale Read. This avoids the situation where the specified timestamp that used by TiDB goes ahead of the latest TSO allocating progress (such as a timestamp several seconds ahead), or goes behide of the GC safe point timestamp. When the specified timestamp goes beyond the service scope, TiDB returns an error or waits for the transaction to commit.
```

**Review by NingLin-P — COMMENTED** [review]
Rest LGTM

File: as-of-timestamp.md [hunk 2]
```diff
@@ -32,7 +36,13 @@ Here are some examples of the `AS OF TIMESTAMP` clause:
...
 - `AS OF TIMESTAMP TIDB_BOUNDED_STALENESS(NOW() - INTERVAL 20 SECOND, NOW())`: Tells TiDB to read the data as new as possible within the time range of 20 seconds ago to the present.
 
-Note that in addition to specifying a timestamp, the most common use of the `AS OF TIMESTAMP` clause is to read data that is several seconds old. If this approach is used, it is recommended to read historical data older than 5 seconds.
+> **Note:**
+>
+> In addition to specifying a timestamp, the most common use of the `AS OF TIMESTAMP` clause is to read data that is several seconds old. If this approach is used, it is recommended to read historical data older than 5 seconds.
+>
+> You need to deploy the NTP service for your TiDB and PD nodes when you use Stale Read. This avoids the situation where the specified timestamp used by TiDB goes ahead of the latest TSO allocating progress (such as a timestamp several seconds ahead), or goes behind the GC safe point timestamp. When the specified timestamp goes beyond the service scope, TiDB returns an error or waits for the transaction to commit.
```
Comment (CharLotteiu) [inline as-of-timestamp.md:43]:
```suggestion
> You need to deploy the NTP service for your TiDB and PD nodes when you use Stale Read. This avoids the situation where the specified timestamp used by TiDB goes ahead of the latest TSO allocating progress (such as a timestamp several seconds ahead), or is later than the GC safe point timestamp. When the specified timestamp goes beyond the service scope, TiDB returns an error or waits for the transaction to commit.
```
"behind" is a little bit ambiguous.. "超前于"是指比 GC safe point 时间戳要晚是吗？@NingLin-P 
能不能用 earlier 和 later 这种比较明确的表达呢？

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM

**Review by CharLotteiu — APPROVED** [review]

File: as-of-timestamp.md (hunk 2 above)
Comment (NingLin-P) [reply as-of-timestamp.md:43]: `later` LGTM

**Pull request merged**